action/action.go

```diff
@@ -168,3 +180,15 @@ func execCommand(args ...string) string {
 	}
 	return string(stdout.Bytes())
 }
+
+// importSecScanMetadata reads the security scan from file and returns
+// it b64encoded.
+func importSecScanMetadata() string {
+	secScanFilePath := ".release/security-scan.hcl"
```

Comment:
Nit: I usually try to make these `const`, it signals to the reader that this is a static value and not expected to change.  I believe the compiler will also do some optimizations, though for this use case that isn't terribly important.